kibana-reports/server/model/index.ts

```diff
@@ -14,30 +14,33 @@
  */
 
 import { schema, TypeOf } from '@kbn/config-schema';
+import {
+  REPORT_TYPE,
+  TRIGGER_TYPE,
+  FORMAT,
+  SCHEDULE_TYPE,
+  REPORT_STATE,
+  REPORT_DEFINITION_STATUS,
+} from '../routes/utils/constants';
 
 export const dataReportSchema = schema.object({
+  ref_url: schema.uri(),
```

Comment:
np: can we use the full word instead of `ref` shorthand?

Reply:
use `base_url` instead